### PR TITLE
ZK-4777: signature can have a background image

### PR DESCRIPTION
**Pull request by b952426**
ZK-4777: signature can have a background image

### CODE REVIEW

File: zktest/src/archive/test2/F96-ZK-4777.zul [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		2. signature1 background color is blue, signature2 is black.
+		3. draw and click clear button in the tool bar, background color and image should not be cleared.
+		4. draw and click save button in the toolbar.
+		5. for signature1, only the drawing should be saved(background image should not be saved).
+		6. for signature2, both the drawing and background should be saved.
+	</label>
+		<signature id="signature1" width="600px" height="250px" onSave="image.setContent(event.media)"
+			backgroundImage="/zktest/test2/img/F96-ZK-4777-frame.png" backgroundIncluded="false" backgroundColor="#b3e0ff"/>
+		<signature id="signature2" width="600px" height="250px" onSave="image.setContent(event.media)"
+			backgroundImage="/zktest/test2/img/F96-ZK-4777-frame.png" backgroundColor="null" penColor="white"/>
```
Comment (scribetw) [inline zktest/src/archive/test2/F96-ZK-4777.zul:27]: try mvc way to pass `null`

**Review by scribetw — CHANGES_REQUESTED** [review]
use a free image instead

**Review by scribetw — APPROVED** [review]

**Review by plijyfes — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**